app/views/layouts/application.html.erb

```diff
@@ -18,14 +18,15 @@
       padding-left: 20px;
       color: #D8000C;
     }
+    body{ background-color: #f9fbff;}
     </style>
   </head>
 
   <body>
 
       <nav>
         <div class="nav-wrapper" style="background-color: #0032bc">
-          <a href="http://css.uwindsor.ca/" class="brand-logo" style="padding: 0 0 0 20px">Computer Science Society</a>
+          <a href="http://localhost:3000/events" class="brand-logo" style="padding: 0 0 0 20px">CSS Events</a>
```

Comment:
```suggestion
          <a href="/events" class="brand-logo" style="padding: 0 0 0 20px">CSS Events</a>
```
This would link the user to localhost in production 😛

Reply:
Oh yeah I didn't know what to write there